_monitoring-plugins/alerting/security.md

```diff
@@ -31,6 +31,9 @@ Later, the user `psantos` wants to edit the monitor to run every two hours, but
 
 After making the change, the monitor now runs with the same permissions as `psantos`, including any [document-level security]({{site.url}}{{site.baseurl}}/security-plugin/access-control/document-level-security/) queries, [excluded fields]({{site.url}}{{site.baseurl}}/security-plugin/access-control/field-level-security/), and [masked fields]({{site.url}}{{site.baseurl}}/security-plugin/access-control/field-masking/). If you use an extraction query to define your monitor, use the **Run** button to ensure that the response includes the fields you need.
 
+Once a monitor is created, the Alerting plugin will continue executing the monitor, even if the user who created the monitor has permissions removed. To stop a monitor, a user with the correct cluster permissions must manually disable or delete the monitor. To disable a monitor, the user must have `cluster:admin/opendistro/alerting/monitor/write`, and to delete a monitor, the user must have `cluster:admin/opendistro/alerting/monitor/delete`. This rule applies to all types of monitors.
```

Comment:
There's a comma splice in the sentence, "To disable a monitor...". Switching the permissions to a short unordered list might improve readability. For example:

```suggestion
Once a monitor is created, the Alerting plugin will continue executing the monitor, even if the user who created the monitor has their permissions removed. A user can only manually delete or disable a monitor with the correct cluster permissions. This rule applies to all types of monitors. 

- To disable a monitor, the user must have the `cluster:admin/opendistro/alerting/monitor/write` permission.
- To delete a monitor, the user must have the `cluster:admin/opendistro/alerting/monitor/delete` permission.
```

You can also remove the comma splice with a period.